src/renderer/containers/Header/index.tsx

```diff
@@ -16,11 +16,11 @@ import React from "react";
 import { useToasts } from "react-toast-notifications";
 
 import { PlayIcon } from "@/components/PlayIcon";
+import { assertPlayKey } from "@/lib/gql";
```

Comment:
`gql.ts` is not a great name for a library. GraphQL is the method of fetching data but doesn't describe what the library actually does. I would prefer the file to be called `slippi.ts` or something because it interacts with the slippi backend. Whether it uses REST or GraphQL or websockets or IPC to do the changes doesn't matter so it shouldn't be named as such.